bioblend/_tests/template_galaxy.ini

```diff
@@ -21,3 +21,4 @@ enable_beta_workflow_modules = True
 master_api_key = $BIOBLEND_GALAXY_MASTER_API_KEY
 enable_quotas = True
 cleanup_job = onsuccess
+enable_notification_system = true
```

Comment:
Guess the ini is only needed for tests against old Galaxy versions which do not support notifications anyway?

Reply:
Makes sense. I removed it

Reply:
I also removed it from the galaxy_ini.yml. I guess that was an issue. Also the failed test (test_update_broadcasted_notification) lacked the following condition (which I added now):

```
  if not self.gi.config.get_config()["enable_notification_system"]:
      self.skipTest("This Galaxy instance is not configured to use notifications.")
  if not self.gi.users.get_current_user()["is_admin"]:
      self.skipTest("This tests requires the current user to be an admin, which is not the case.")
```
As such it was not skipped and failed with the following error:
`Unexpected HTTP status code: 403: {"err_msg":"Notifications are disabled in this Galaxy.","err_code":403004}`

I added `enable_notification_system = true` to the template_galaxy.yaml file again. The tests now succeed locally.

I would be grateful if you could explain why the condition is required in the .yaml file but not in the .ini

Reply:
> I would be grateful if you could explain why the condition is required in the .yaml file but not in the .ini

I was just **guessing** that the ini file is only used for older Galaxy versions which do not support yaml config files. .. And these old versions do not support the notification system anyway.

If I'm right it should not change anything when removed from the ini file...

Reply:
It works now. Thanks for the suggestion :)